x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/common/validation/SourceDestValidatorTests.java

```diff
@@ -260,17 +274,17 @@ public void testCheck_GivenMixedMissingAndExistingConcreteSourceIndex() throws I
             }
         );
 
-        assertValidation(
-            listener -> simpleNonRemoteValidator.validate(
-                CLUSTER_STATE,
-                new String[] { SOURCE_1, "missing" },
-                "dest",
-                SourceDestValidator.NON_DEFERABLE_VALIDATIONS,
-                listener
-            ),
-            true,
-            null
-        );
+//        assertValidation(
+//            listener -> simpleNonRemoteValidator.validate(
+//                CLUSTER_STATE,
+//                new String[] { SOURCE_1, "missing" },
+//                "dest",
+//                Collections.emptyList(),
+//                listener
+//            ),
+//            true,
+//            null
+//        );
```

Comment:
^ needs re-activation or should be removed

Reply:
Reactivated it. Good catch.